server/src/main/java/timely/store/iterators/KeyValuePair.java

```diff
@@ -0,0 +1,36 @@
+package timely.store.iterators;
+
+import org.apache.accumulo.core.data.Key;
+import org.apache.accumulo.core.data.Value;
+
```

Comment:
So, it appears that KeyValue creates a new Value object instead of being a holder for it.